packages/salesforcedx-vscode-core/src/context/workspaceOrgType.ts

```diff
@@ -25,47 +24,28 @@ export enum OrgType {
  */
 export async function getWorkspaceOrgType(): Promise<OrgType> {
   const workspaceContext = WorkspaceContext.getInstance();
-  const connection = await workspaceContext.getConnection();
+  let connection;
+  try {
+    connection = await workspaceContext.getConnection();
+  } catch (error) {
+    console.warn(
+      `An error was encountered while getting the org connection:\n ${error}`
+    );
+    return OrgType.NonSourceTracked;
```

Comment:
If we can't return the org connection, why do we assume we have a NonSourceTracked org?

Reply:
Its basically a default value for this function to return.  When set to NonSrcTracked, commands like view changes and push/pull are hidden.